scriptRunner/src/keybase.js

```diff
@@ -0,0 +1,48 @@
+const fetch = require("node-fetch")
+const database = require("../../api/lib/database")
+const config = require("../../api/config")
+const db = database(config)
+
+module.exports.getKeybaseImages = async () => {
+    const networks = await db("").getNetworks()
+    const cosmosNetworks = networks.filter(({network_type}) => network_type === "cosmos")
+    cosmosNetworks.forEach(async network => {
+        const schema = network.id.replace(/-/g, "_")
+        const validators = await db(schema).read(
+            `validatorprofiles`,
+            `validatorprofiles`,
+            ['operator_address', 'name', 'last_updated', 'profile_identifier']
+          )
+        const updateableValidators = validators
+            .filter(({profile_identifier}) => profile_identifier)
+            .filter(({last_updated}) => {
+                // only update validators every 24h
+                return Date.now() - new Date(last_updated).getTime() > 1000 * 60 * 60 * 24
+            })
+        const updatedValidators = await Promise.all(
+            updateableValidators
+            .map(async ({ operator_address, profile_identifier, name }) => {
+                const query = `https://keybase.io/_/api/1.0/user/user_search.json?q=${profile_identifier}&num_wanted=1`
+                const result = await fetch(query).then(res => res.json())
+                if (result.list.length > 0) {
+                    return {
+                        name,
+                        operator_address,
+                        picture: result.list[0].keybase.picture_url
+                    }
```

Comment:
Is it not possible to just store the images in DB? Or are they so dynamic? Don't think so, right?

Reply:
ah ok, only update if we see they updated their profile. Nice, slick!

Reply:
What is the problem you want to solve?